src/System.Security.Permissions/tests/System.Security.Permissions.Tests.csproj

```diff
@@ -1,29 +1,30 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
```

Comment:
I don't think you need this, as it looks like the test assembly should be AnyCPU